cmd/ddev/cmd/utils.go

```diff
@@ -60,3 +61,27 @@ func getRequestedProjects(names []string, all bool) ([]*ddevapp.DdevApp, error)
 
 	return requestedProjects, nil
 }
+
+// commandRequiresDocker are commands that can be run without a Docker runtime.
+func commandRequiresDocker(commands []string) bool {
+
+	// List of commands that do not rely on Docker running.
+	allowedCommands := []string{
+		"version",
+		"-v",
+		"--version",
+		"--help",
+		"-h",
+		"get",
+		"hostname",
+		"list",
+		"config",
```

Comment:
I guess `service`  (disable and enable) would go here... I would have expected `delete` but guess it requires docker running.

Reply:
I updated the code to always check for Docker.